index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=`, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/app.css">
+    <title>CV</title>
+  </head>
+  <body>
+    <div class="page-container">
+      <header class="container"><img class="user-picture" src="./images/avatar.jpg" alt="The picture of Dmitriy Kunets" width="250" height="250">
+        <h1 class="user-name">Kunets Dmitriy</h1>
+      </header>
+      <main class="flex-container">
+        <section class="flex-block-6">
+          <div class="block-title-wrapper"><i class="fa fa-user"></i>
+            <h1 class="title">Summary</h1>
+          </div>
+          <div class="block-content">
+            <p>
+              I like to learn something new both in web technologies and in programming. 
+              My goal is to learn something new and improve my skills in JavaScript and JS frameworks. 
+              It is interesting for me and I am ready to spend a lot of time on it.
+            </p>
+            <p>Also I love to participate in hackathons, it is amazing to do something crazy for 48 hours.</p>
+          </div>
+        </section>
+        <section class="flex-block-6">
+          <div class="block-title-wrapper"><i class="fa fa-id-badge"></i>
+            <h1 class="title">Profile</h1>
+          </div>
+          <div class="block-content">
+            <table class="profile-table" cellpadding="7">
```

Comment:
I recommend to use flex or grid layout instead table, because this data doesn't look like real table data. More about [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox) and [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)